sdk/containerregistry/tests.yml

```diff
@@ -9,6 +9,15 @@ stages:
       MatrixReplace:
         - TestSamples=.*/true
       DeployArmTemplate: true
+      Clouds: 'Public,UsGov,China,Canary'
```

Comment:
You can delete all these added lines, and replace with:

`SupportedClouds: 'Public,UsGov,China,Canary'`

I'd suggest removing canary as well unless you specifically need it.